src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,37 @@
+package qna.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import qna.exception.CannotDeleteException;
+
+public class Answers {
+
+    public static final String CANNOT_DELETE_SOMEONE_ELSE_ANSWER = "다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.";
+    private final List<Answer> answers;
+
+    public Answers(Answer... answers) {
+        this.answers = Arrays.asList(answers);
+    }
+
+    public Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public DeleteHistories deleteAnswers(User loginUser) throws CannotDeleteException {
```

Comment:
```suggestion
    public DeleteHistories deleteAnswers(User loginUser) {
```

외부에서 강제로 예외를 처리하도록 만들 필요는 없어보이네요

Reply:
아~
CannotDeleteException 을 RuntimeException으로 변경하면서 컴파일오류가 이제 발생하지 않는군요.👍